### ISSUE TITLE
Display the current running time entry (if any)

### ISSUE THREAD

**Issue by alexdewar** [alexdewar/clockify-tui#5]
We want to show the user what the current running time entry is in the app. If there is no running time entry, we should display `(none)` or something similar.

**Comment by Sahil590** [comment]
I'll have a look at this one

**Comment by alexdewar** [comment]
@Sahil590 Are you planning on working on this today? No pressure!

**Comment by Sahil590** [comment]
To get a list of tasks, we would need the project ID. 
#27 will handle this

**Comment by Sahil590** [comment]
So with these tasks it not really something that is useful to this tool since we don't use them. After looking at the api docs, I think we need to work more with the time entry and this would align more with what we want to do

**Comment by alexdewar** [comment]
Yep, it seems like it. In retrospect, I *meant* "time entries" rather than "tasks" -- I didn't realise they were different things. I've amended the issue now.
